### ISSUE TITLE
Adjustable column size

### ISSUE THREAD

**Issue by cyberfeign** [aome510/spotify-player#278]
**Is your feature request related to a problem? Please describe.**
I use this awesome client in a Tmux pane on a custom panel with other applications where space is rather scarce. I've noticed that for instance track duration is sometimes clipped when dimensions are reduced.

**Describe the solution you'd like**
I'd like to be able to configure the width/number of characters per column (independently?) in browse view etc. so that I can adjust space according to preference and avoid content outside viewable area.

**Describe alternatives you've considered**
Perhaps also adjusted settings for automatic column condensation depending on content and window size. Certain columns sometimes have unused space while other columns are too narrow or outside window area.

**Comment by aome510** [comment]
The clipped `duration` column seems to be because not all columns in the track table are constructed using percentage constraint:

https://github.com/aome510/spotify-player/blob/3b2a36625ac5978e8e68cb662bad340e15cc29a4/spotify_player/src/ui/page.rs#L743-L750

Regarding the adjustable column size, do you mean able to configure the percentage of each column? For example,
```toml
title = 25
artists = 30
```

**Comment by Autobeing** [comment]
Precisely, something like that is what I had in mind. Percentages would make it easy to understand, with a clear default value as well.

**Comment by cyberfeign** [comment]
Yep, that looks good. Odd that it's still clipping, maybe due to how the width of the window is set?

**Comment by apprehensions** [comment]
Please! this damn padding at the bottom of my spotify_player has been bothering me for too long
